velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +91,24 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseWeibullCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double a, double b, double p) {
+    constexpr double kInf = std::numeric_limits<double>::infinity();
+
+    VELOX_USER_CHECK_GT(a, 0, "a must be greater than 0");
+    VELOX_USER_CHECK_GT(b, 0, "b must be greater than 0");
+    VELOX_USER_CHECK((p >= 0) && (p <= 1), "p must be in the interval [0, 1]");
+
+    if ((a == kInf) || (b == kInf)) {
+      result = 0.0;
+    } else {
+      result = b * std::pow(-std::log1p(-p), 1.0 / a);
```

Comment:
What is the origin of this formula ? 

Does Boost not have a Weibull distribution function that can be used here ?

Reply:
Boost does have a Weibull CDF function, but it doesn't have direct function for available for Inverse Weibull CDF. I referred Java definition which uses this Java-equivalent formula.